### ISSUE TITLE
question about the loss function

### ISSUE THREAD

**Issue by wangxw5** [lephong/mulrel-nel#15]
in your loss function, you use
`X = F.normalize(self.rel_embs)
        diff = (X.view(self.n_rels, 1, -1) - X.view(1, self.n_rels, -1)).pow(2).sum(dim=2).add_(1e-5).sqrt()
        diff = diff * (diff < 1).float()
        loss -= torch.sum(diff).mul(lamb)`
what do they means? why you use them in the loss function?
thanks a lot.
